style.css

```diff
@@ -64,18 +68,18 @@ li {
   height: 80vh;
   overflow: hidden;
   color: rgb(20, 3, 3);
+
   /* hide the menu above the screen by default */
   transform: translateX(-100%);
   transition: transform 0.2s ease;
-  background-image: url("/assert/image_geometry_menu_1.svg");
+  background-image: url("assert/image_geometry_menu_1.svg");
   background-repeat: no-repeat;
-
 }
 
 .showNav {
```

Comment:
## OPTIONAL 

- [ ] Please try your best to fix this. 
- Your design
![m](https://user-images.githubusercontent.com/63972670/167686628-29fe0907-aeaf-4226-8d92-e80ca29d09fe.PNG)
 - Original Design

![n](https://user-images.githubusercontent.com/63972670/167687200-7b30536a-bc3f-45af-9108-f8fa03145e6d.PNG)